### ISSUE TITLE
Hello, thank you very much for your paper and code demo. I have a few questions that I would like to ask.

### ISSUE THREAD

**Issue by hxc236** [NKUhealong/BSNet#1]
I'm a student studying and reproducing your paper. If it's convenient for you, I'd love to be able to ask you for some specific details about training.

1. What does `mutual.py` do? At first I thought it was the DML training code, but it didn't seem to be the case.

2. How did you preprocess the DRIVE dataset? In the DRIVE dataset, only the training set has manual annotation labels, but the test set does not. How should we calculate test_acc during training and MAE, Dice, IoU and other indicators during testing?

3. How is your DML method applied in SSL, what is the model_arch of the two student networks, and how should the loss be calculated? If possible, can you provide a copy of the code? 

I'm new to this field and the questions may be dumb and complex, so thank you very much for your time and help!
